### ISSUE TITLE
/change/card returns html code

### ISSUE THREAD

**Issue by bluekite2000** [pinax/pinax-stripe#13]
Instead of /change/card it becomes /a/change/card/ with the following output ( i have added bootstrap-ajax.js but doesnt help) : 

{"html": "\n<div class=\"subscribe-form\">\n\t<p >\n\t    \n\t        Your currently have the <strong>Calling VIP ($50/month)</strong> plan.\n\t    \n\t</p>\n\t  <p >\n\t      \n\t          Current card on file is a <strong>Visa</strong>\n\t          ending in the digits <strong>4242</strong>.\n\t      \n\t  </p>\n\n\t      \n    <form action=\"/a/subscribe/\" class=\"form-horizontal ajax\" data-replace-closest=\".subscribe-form\" method=\"POST\">\n        <div style='display:none'><input type='hidden' name='csrfmiddlewaretoken' value='bFBIQlWa8wGPSdFOS8Mph3300hUV05lg' /></div>\n        \n\n\t<fieldset>\t\n\n\t\t\n\t\n\t\t\t<div class=\"clearfix\">\n\t\t\t\t<label for>  <label for=\"id_plan\">Plan</label></label>\n       \t\t\t<div class=\"input\"> <select name=\"plan\" id=\"id_plan\">\n<option value=\"monthly1\">Calling Lite ($10/month)</option>\n<option value=\"monthly3\">Calling VIP ($50/month)</option>\n<option value=\"monthly2\">Calling Pro ($25/month)</option>\n<option value=\"\" selected=\"selected\">-------</option>\n</select></div>\t\t\t\n         \t</div>       \n        \n\n\t</fieldset>\n\n\n        <div class=\"control-group\">\n            <label class=\"control-label required-field\">\n                Credit card\n            </label>\n\n\t\t        <div style='display:none'><input type='hidden' name='csrfmiddlewaretoken' value='bFBIQlWa8wGPSdFOS8Mph3300hUV05lg' /></div>\n\t\t        <input type=\"hidden\" name=\"stripeToken\">\n\t\t        <script src=\"https://button.stripe.com/v1/button.js\" class=\"stripe-button\" data-key=\"pk_test_nqD3mDAvmbrIAeZSRELkwoK6\"></script>\n    </form>\n</div>\n\n", "location": "/history/"}
